Podfile.lock

```diff
@@ -80,7 +80,7 @@ PODS:
   - MRProgress/ProgressBaseClass (0.7.0)
   - MRProgress/Stopable (0.7.0):
     - MRProgress/Helper
-  - NewRelicAgent (4.155)
+  - NewRelicAgent (4.174)
```

Comment:
@aerych - you should do `pod install` instead of pod update as it'll just get down the pods you need instead of forcing an update prematurely. Even these small incremental changes can introduce bugs.